lib/publiccloud/provider.pm

```diff
@@ -354,16 +354,19 @@ sub create_instances {
         }
         # check guestregister conditional, default yes:
         $instance->wait_for_guestregister() if ($args{check_guestregister});
+
         # Performance data: boottime
-        if (!is_openstack && is_ok_url($url)) {
```

Comment:
but this line already suppose to skip openstack

Reply:
indeed, the `else` branch will be selected in such case,therefore either we accept that `continue` or we'd remove it and append the `... if not openstack` to the else code.

Reply:
Thanks, Maurizio, for the clarification.

Reply:
Another convenence in `continue` is that the full `is_ok_url`  execution, to complete te condition evaluation, will be skipped for openstack, indeed saving that check time.

Reply:
Point clarified in a huddle, resolving